examples/serialrx/serialrx_main.c

```diff
@@ -118,8 +118,7 @@ int main(int argc, FAR char *argv[])
   while (cnt < bytecount)
     {
 #ifdef CONFIG_EXAMPLES_SERIALRX_BUFFERED
-      size_t n = fread(buf, 1, 26, f);
```

Comment:
why make the follow change

Reply:
I'm very sorry, this change is really optional, this 26 I double-checked a few times without finding any clues, then I changed it to a simple number. Maybe it helps.

Reply:
Ok, but after you finish the investigation is better to remove the temp change. If you think the temp change is worth to upstream, please split to the different patch. It's important to ensure one patch fix one problem, so the reviewer could understand your change quickly.

Reply:
Okay, I see what you mean.I'd still like to ask, but would these small changes be a waste of your time.After all, it doesn't change the logic of the code

Reply:
You can have one PR which contain more than one patch